corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js

```diff
@@ -556,9 +556,9 @@ cloudCare.AppView = Backbone.View.extend({
             ).addClass("btn btn-primary").appendTo(
                 self.formListView.caseView.detailsView.el
             );
-            $('<a />').attr('href', buttonUrl).attr('target', '_blank').text(translatedStrings.newWindow).appendTo(
+            $('<a class="btn btn-default"/>').attr('href', buttonUrl).attr('target', '_blank').text(translatedStrings.newWindow).appendTo(
```

Comment:
i think we actually do use this for opening a link in a new tab, so in my mind it makes sense to leave it an anchor rather than a button with an event that opens a link